iMessage/dumpMessages.js

```diff
@@ -58,7 +58,7 @@ function pm(p) {
 }
 
 // Offset from macOS 10.14.3
```

Comment:
```suggestion
// Offset from macOS 30.21.11
```

Reply:
Bro do you know how change offset value. Coul you tell me

Reply:
I try to contact evey developer no one reply me. I need help